### DISCUSSION TITLE
Support org-level workflows

### DISCUSSION THREAD

**Discussion by suhas-coursera** [community/community#72759] [Actions]
### Select Topic Area

Product Feedback

### Body

We have multiple use cases to trigger a common Github action workflow for the same event across 100's of repos, Currently, we're using a reusable workflow but still, it involves the creation of workflow in all 100 repos to call the reusable one, Instead, it'll be convenient to create the workflow at org-level and enable it for all or selected repos.

**Comment by juliandunn** [comment]
Have you tried using the ruleset system with [required workflows](https://github.blog/changelog/2023-10-11-requiring-workflows-with-repository-rules-is-generally-available/)?

**Reply by suhas-coursera** [reply]
It only helps in ensuring workflow pass or fail across multiple repos by configuring just at org-level, But I am looking for a one-place solution for reusable workflows, where workflows can be created at the org-level and apply it for all/selected repos.

**Reply by juliandunn** [reply]
I'm not understanding the difference. Can you give an example of what those workflows are doing? Are they on other triggers for example other than `push` or `pull_request`/`pull_request_target`?
